BlazarUI/app/scripts/models/Log.js

```diff
@@ -14,8 +13,27 @@ class Log extends Model {
   }
 
   url() {
-    return this.logUrl;
+    return `${config.apiRoot}/build/${this.buildNumber}/log/`;
   }
+
+  formatLog(jqxhr) {
+    let cache = [];
+
+    if (jqxhr.status !== 200) {
+      console.warn(jqxhr);
+      return "<p class='x-y-roomy'>Error loading build log. Please check your console for more detail.</p>";
+    }
+
+    const lines = jqxhr.responseJSON.data.split('\n');
+
+    lines.forEach( (line) => {
+      cache.push(`<p class='log-line'>${line}</p>`);
+    });
+
+    return cache.join('\n');
```

Comment:
I see what was going on.  I had the `log-lines` set as inline-block, with the container as a pre tag, the join with `\n` was fixing the issue of them being displayed inline. Dropped the inline-block and it's all set. Thanks for noticing that.